Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding-sdk
+
+The Scene Understanding observer inherits from older spatial awareness concepts so the implementation is and odd fit in places.
+
+Eventually this system should replace the existing SpatialAwareness observer.
+
+Scene Understanding is a __platform specific__ technology. Spatial Awareness is meant to cover _platform agnostic_ capabilities. With few data points, it's likely we don't have the proper abstraction yet. 
+
+## Prerequisites
+
+Ensure build settings' platform is set to UWP.
+
+Ensure your project is configured to use MSBuild.
+
+You may need to restart unity if you get errors about missing assembly references. This should trigger download of packages.
+
+![Manage MRTK Configuration](Images/ConfigureMRTKMSBuildMenu.jpg)
+
+![MSBuild enabled](Images/MRTKProjectConfiguratorMSBuildEnabled.jpg)
+
+
+## Demonstration and debugging SU
+
+### Unity demo scene
+
+Open the scene file and press play!
+
+`Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/DemoSpatialAwareness.unity`
+
+#### Configuring the observer service
+
+Select the 'MixedRealityTookit' game object and check the inspector.
+
+![scene understanding location in heirarchy](Images/MRTKHierarchy.png)
+![mrkt location in inspector](Images/MRTKLocation.png)
+
+These options will allow you to configure the observer on play. 
+
+If you press play at this point the example serialized scene data will be loaded and displayed in the editor (assuming the observer is configured to show 'debug' objects)
+
+These 'debug' options are meant to demonstrate some of the features of the SU DLL. It is intended that most consumers of SU will write their own scripts that subscribe to SU events, and do whatever they want with the info. In addition many observer options may be set at runtime.
+
+The provided DemoSpatialAwarenessController.cs offers examples of these things.
+
+### Example script
+
+The example script _DemoSpatialAwarenessController.cs_, demonstrates the major concepts in working with the SU service.
+
+* Subscribing to SU events
+* Handling SU events
+* Finding the nearest platform
+* Instantiating a prefab and placing it in an open spot
+
+The script uses MRKT UI to toggle service features on the controller.
+
+Turning on "Instantiate Prefabs", will demonstrate creating objects that size to fit themselves to all Scene Objects, gathered neatly under a parent object.
+
+![demo controller options](Images/Controller.png)
+
+### Built app notes
+
+You may build and deploy to HoloLens in the standard way. Once running, you'll be presented with a number of buttons to play with the features.
+
+The "Place" button will make a prefab appear on the nearest platform. (which may be behind you!) You should hear a spatial sound when this occurs to help you find it.
```

Comment:
In general, it is best to avoid informal you, instead say things like One should hear a spatial sound....etc